frontend-react/src/AppRouter.tsx

```diff
@@ -85,6 +86,10 @@ export const AppRouter = () => {
                 path={"/admin/value-sets"}
                 element={<ValueSetsIndexWithAuth />}
             />
+            <Route
+                path="/admin/revisionhistory/org/:org/settingtype/:settingType"
```

Comment:
[nit] we've already got a route for `/admin/orgnewsetting/org/:orgname/settingtype/:settingtype"` - resistance to using `:orgname` as our segment variable here for consistency?

Reply:
I switched it because you said you didn't like allsmallcaps. I am neutral on naming, so happy to switch it bac,.

Reply:
OK. Switching it has a ripple effect because it's also used in the API call. So, I'm asking to punt on this item. 

I just hated the mixture of `settingType` with `orgname` everywhere. I figured it was better to be locally consistent and eventually fix `orgname` to be `orgName` or `org`

Reply:
yep, forgot about that earlier convo, we can punt